src/ipred.rs

```diff
@@ -1504,7 +1509,8 @@ mod neon {
         ) {
             let dst = dst.cast();
             let topleft = topleft.as_ptr().cast();
-            self.get()(dst, stride, topleft, width, height, dxy, max_base_xy)
+            // SAFETY: We're assuming the asm is actually correct and safe.
```

Comment:
Nit: I prefer the wording you've used previously that specifically notes that the Rust fallback is fully safe and that the asm routine is supposed to do the same. Not super important, I just think it better clarified why we think the function pointer call is safe.

Reply:
But there's no fallback for this.